disperser/apiserver/server.go

```diff
@@ -178,67 +178,86 @@ func (s *DispersalServer) DisperseBlob(ctx context.Context, req *pb.DisperseBlob
 	return reply, err
 }
 
-func (s *DispersalServer) disperseBlob(ctx context.Context, blob *core.Blob, authenticatedAddress string) (*pb.DisperseBlobReply, error) {
-	timer := prometheus.NewTimer(prometheus.ObserverFunc(func(f float64) {
-		s.metrics.ObserveLatency("DisperseBlob", f*1000) // make milliseconds
-	}))
-	defer timer.ObserveDuration()
+func (s *DispersalServer) validateBlobRequest(ctx context.Context, blob *core.Blob) error {
 
 	securityParams := blob.RequestHeader.SecurityParams
 	if len(securityParams) == 0 {
-		return nil, fmt.Errorf("invalid request: security_params must not be empty")
```

Comment:
These validation errors don't currently result in call to `s.metrics.HandleFailedRequest(quorumId, blobSize, "DisperseBlob")`

Reply:
The invalid requests may be just bogus